CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+2023-09-15
+----------
+- Remove start_object_detection and stop_object_detection services
+- Add enable_object_detection service that takes a bool as parameter to enable/disable the OD module (same behavior as in ROS 2 Wrapper)
```

Comment:
Found the following error when using the new service:
```
root@sl-mrousseau:~/catkin_ws# rosservice call /zed2/zed_node/enable_object_detection 0
success: True
message: "Object Detection stopped"
root@sl-mrousseau:~/catkin_ws# rosservice call /zed2/zed_node/enable_object_detection 1
ERROR: service [/zed2/zed_node/enable_object_detection] responded with an error: b''
root@sl-mrousseau:~/catkin_ws# rosservice call /zed2/zed_node/enable_object_detection 1
ERROR: service [/zed2/zed_node/enable_object_detection] responded with an error: b''
```

Reply:
I'll check it. What command did you use to call the service?
With the `rqt` GUI there are no issues

Reply:
The service successfully works and toggles the od on and off, however turning it on returns this error in the cli

Reply:
Fixed